static/js/App.tsx

```diff
@@ -18,18 +18,29 @@ import StorageList from "pages/storages/StorageList";
 import ProfileDetail from "pages/profiles/ProfileDetail";
 import CertificateGenerate from "pages/certificates/CertificateGenerate";
 import { SharedNotifyProvider } from "./context/sharedNotify";
+import NoAuth from "pages/certificates/NoAuth";
 
 const queryClient = new QueryClient();
 
 const App: FC = () => {
+  const [isAuthenticated, setAuthenticated] = useState(false);
```

Comment:
@lorumic , wouldn't it be better to create a context + a custom hook for that?
Like 
```
const { isAuthenticated } = useAuth();
```
You wouldn't need to pass that state down to the components in that case.

Reply:
Maybe, but at this point this information is really only needed by `App` and `Navigation`, not by any other component, so I thought a simple state that is passed to `Navigation` was enough, without introducing too much overhead. Of course, it can be always changed later, if the information needs to be passed elsewhere or if it's considered a bad practice to do it like this. This is just, let's say, a first iteration - the design of the `NoAuth` page is also likely to change a lot.